formats/policy/frontend/schema.json

```diff
@@ -99,6 +99,9 @@
                     }
                   }
                 }
+              },
+              "result_metadata_name_key": {
```

Comment:
Yep, i'm putting that PR together now.

I'd assumed the process was get the schema agreed first - then other PRs (though i cheated with the FF one, as it's got more than just a schema update.